src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -185,7 +234,10 @@ private String commandStringGenerator(
       String pathToScript, String scriptName, String... attributes) {
```

Comment:
@l-marchesi just wanted to bring up the idea, I'll have @jpduloch decide how to proceed with this.